chemprop/data/data.py

```diff
@@ -159,6 +173,18 @@ def features(self) -> List[np.ndarray]:
 
         return [d.features for d in self._data]
 
+    def atom_descriptors(self) -> List[np.ndarray]:
+        """
+        Returns the atom descriptors associated with each molecule (if they exit).
+
+        :return: A list of 2D numpy arrays containing the atom descriptors
+        for each molecule or None if there are no features.
```

Comment:
Tiny detail for the ReadTheDocs documentation generation: if you continue the `:param:` or `:return:` docstring on the next line, it needs to be indented as follows.

```suggestion
        :return: A list of 2D numpy arrays containing the atom descriptors
                 for each molecule or None if there are no features.
```